### ISSUE TITLE
Option for a regular maximized window with title bar

### ISSUE THREAD

**Issue by JFMugen** [Ruben2776/PicView#166]
I would like to have an option for a regular maximized window with the visible title bar. Like this; 

![SS241002171735](https://github.com/user-attachments/assets/c2daf767-7343-46b3-9a93-5227592f6541)

Taskbar would be blurred as well (this is more of a personal setting) if it would be a regular maximized window but the way you set up makes it act like a small window I guess? 

What do you think?

**Comment by Ruben2776** [comment]
I'm sorry, I'm not sure if I'm following you 100%.

If you would like to cover the whole screen, you can enable the function `Stay on top of other windows` and manually resize the window to cover the taskbar. If you're hiding the bottom toolbar, then the Windows taskbar would technically be blurred by the application's background color.

Is this what you meant?

Otherwise, I'm currently working on rewriting the project to a newer platform. You can read more [here](https://github.com/Ruben2776/PicView/issues/159), if you're interested. Any future feature requests would take part there.  
I'm experimenting with the idea of both having a full-screen state, and a maximized state, for a new future version.

**Comment by JFMugen** [comment]
> Otherwise, I'm currently working on rewriting the project to a newer platform. You can read more [here](https://github.com/Ruben2776/PicView/issues/159), if you're interested. Any future feature requests would take part there. I'm experimenting with the idea of both having a full-screen state, and a maximized state, for a new future version.

Yeah I made it a bit confusing for no reason. Taskbar thing is about my settings, it is clean when there is no maximized window and blurred when there is one, so it doesn't work well with PicView's lack of maximized state. 

So basically this request is 'maximized state' like you said. I installed Avolonia version and I will follow the project, thanks ^^

**Comment by Ruben2776** [comment]
With the release of version 3.0.2 I've fixed the maximized margin issue, added the `Maximize` function to the `Keybindings` window, as well as adding it to the context menu when right-clicking. 

Does this solve your issue?

I might, at a later date, add an option to chose if the button, between close and minimize, should maximize the window or go to fullscreen state.

**Comment by JFMugen** [comment]
Yep, thats what I was asking for. I guess you might go for that option depends on demand, some ppl may prefer fullscreen approach after all. 

Thanks!
